packages/blocks/src/services.ts

```diff
@@ -2,20 +2,20 @@ import type { UnionToIntersection } from '@blocksuite/global/types';
 
 import type { BaseService } from './__internal__/service/index.js';
 import { BookmarkBlockService } from './bookmark-block/index.js';
-import { CodeBlockService } from './code-block/index.js';
+import { LegacyCodeBlockService } from './code-block/index.js';
 import { DatabaseBlockService } from './database-block/database-service.js';
 import { DividerBlockService } from './divider-block/index.js';
 import { ImageBlockService } from './image-block/index.js';
-import { ListBlockService } from './list-block/index.js';
+import { LegacyListBlockService } from './list-block/index.js';
 import type { Flavour } from './models.js';
 import { NoteBlockService } from './note-block/index.js';
-import { ParagraphBlockService } from './paragraph-block/index.js';
+import { LegacyParagraphBlockService } from './paragraph-block/index.js';
 
 export const blockService = {
-  'affine:code': CodeBlockService,
+  'affine:code': LegacyCodeBlockService,
   'affine:database': DatabaseBlockService,
-  'affine:paragraph': ParagraphBlockService,
-  'affine:list': ListBlockService,
+  'affine:paragraph': LegacyParagraphBlockService,
```

Comment:
What's the difference between legacy and non-legacy providers?

Reply:
The legacy service is not part of block spec and will be replaced by the new service in a progressive way.